rubygem-foreman_remote_execution_core/rubygem-foreman_remote_execution_core.spec

```diff
@@ -24,7 +24,17 @@ BuildArch: noarch
 Provides: %{?scl_prefix}rubygem(%{gem_name}) = %{version}
 
 %description
-Use the Remote Execution SSH inside Foreman smart proxy
+Ssh remote execution provider code sharable between Foreman and Foreman-Proxy
+
+%package proxy
+BuildArch:  noarch
+Requires:   %{?scl_prefix}%{pkg_name} = %{version}-%{release}
+Requires:   %{?scl_prefix}rubygem(smart_proxy_dynflow_core) >= 0.1.5
+Obsoletes:  %{?scl_prefix}rubygem(smart_proxy_remote_execution_ssh_core) < 0.1.4
+Summary:    Foreman proxy intergration package for rubygem-%{gem_name}
+
+%description proxy
```

Comment:
While working on packaging smart_proxy_ansible, I'm thinking about simplifying the approach, and putting this directly into `smart_proxy_remote_execution_ssh` rather than having a sub-package here